FileStorage#save causes IllegalFormatConversionException when info logging is enabled

In case when INFO log level is enabled, any FileStorage#save operation fails with the following stack trace:

```
java.util.concurrent.ExecutionException: java.util.IllegalFormatConversionException: d != [B

	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
	at com.artipie.asto.StorageTest.savesAndLoads(StorageTest.java:67)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:230)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:58)
Caused by: java.util.IllegalFormatConversionException: d != [B
	at java.base/java.util.Formatter$FormatSpecifier.failConversion(Formatter.java:4426)
	at java.base/java.util.Formatter$FormatSpecifier.printInteger(Formatter.java:2938)
	at java.base/java.util.Formatter$FormatSpecifier.print(Formatter.java:2892)
	at java.base/java.util.Formatter.format(Formatter.java:2673)
	at java.base/java.util.Formatter.format(Formatter.java:2609)
	at java.base/java.lang.String.format(String.java:2897)
	at com.jcabi.log.Logger.format(Logger.java:128)
	at com.jcabi.log.Logger.infoForced(Logger.java:247)
	at com.jcabi.log.Logger.info(Logger.java:232)
	at com.artipie.asto.fs.FileStorage.lambda$save$6(FileStorage.java:120)
	at io.reactivex.rxjava3.internal.operators.completable.CompletableFromAction.subscribeActual(CompletableFromAction.java:35)
	at io.reactivex.rxjava3.core.Completable.subscribe(Completable.java:2279)
	at io.reactivex.rxjava3.internal.operators.single.SingleFlatMapCompletable$FlatMapCompletableObserver.onSuccess(SingleFlatMapCompletable.java:91)
	at io.reactivex.rxjava3.internal.operators.single.SingleMap$MapSingleObserver.onSuccess(SingleMap.java:64)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableToListSingle$ToListSubscriber.onComplete(FlowableToListSingle.java:102)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableFromArray$ArraySubscription.fastPath(FlowableFromArray.java:138)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableFromArray$BaseArraySubscription.request(FlowableFromArray.java:88)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableToListSingle$ToListSubscriber.onSubscribe(FlowableToListSingle.java:83)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableFromArray.subscribeActual(FlowableFromArray.java:38)
	at io.reactivex.rxjava3.core.Flowable.subscribe(Flowable.java:14816)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableToListSingle.subscribeActual(FlowableToListSingle.java:56)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:3693)
	at io.reactivex.rxjava3.internal.operators.single.SingleMap.subscribeActual(SingleMap.java:34)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:3693)
	at io.reactivex.rxjava3.internal.operators.single.SingleFlatMapCompletable.subscribeActual(SingleFlatMapCompletable.java:44)
	at io.reactivex.rxjava3.core.Completable.subscribe(Completable.java:2279)
	at io.reactivex.rxjava3.core.Completable.subscribe(Completable.java:2353)
	at hu.akarnokd.rxjava3.jdk8interop.CompletableInterop.lambda$await$1(CompletableInterop.java:46)
	at io.reactivex.rxjava3.core.Completable.to(Completable.java:2570)
	at com.artipie.asto.fs.FileStorage.save(FileStorage.java:128)
	at com.artipie.asto.StorageTest.savesAndLoads(StorageTest.java:60)
````

--- a/pom.xml
+++ b/pom.xml
@@ -100,6 +100,13 @@ SOFTWARE.
       <artifactId>commons-io</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-simple -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>1.7.21</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
   <build>
     <plugins>

--- a/src/main/java/com/artipie/asto/fs/FileStorage.java
+++ b/src/main/java/com/artipie/asto/fs/FileStorage.java
@@ -120,7 +120,7 @@ public final class FileStorage implements Storage {
                             Logger.info(
                                 this,
                                 "Saved %d bytes to %s: %s",
-                                bytes, key, target
+                                bytes.length, key, target
                             );
                         }
                     )

